CMakeLists.txt

```diff
@@ -62,6 +62,7 @@ built some, drivers needed by the regression tests will still be built."
   OFF
 )
 
+option(PARTHENON_ENABLE_GPU_MPI_CHECKS "Checks if possible that the mpi num of procs and the number of gpu devices detected are appropriate." ${BUILD_TESTING})
```

Comment:
Can we put a line break here? The comment is long enough it's going to word wrap in text editors.